components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -216,17 +216,13 @@ open class PlacesConnection internal constructor(connHandle: Long, uniffiConn: U
     // Each method here will use one or the other, depending on whether it's been uniffi'd or not.
     protected var handle: AtomicLong = AtomicLong(0)
     protected var conn: UniffiPlacesConnection
-    protected var interruptHandle: InterruptHandle
+    protected var interruptHandle: SqlInterruptHandle
```

Comment:
I have a very vague preference for just `InterruptHandle` here, because it's (a) more than just sql and (b) kinda leaks an implementation detail out (eg, ultimately "tabs" should expose an interrupt handle too, but it doesn't even use sql!)

OTOH, I'm really just saying I hate the name in the rust code, but the UDL having the same name as the rust code does make some sense.

So ultimately I'll leave that up to you.

Reply:
hmm unfortunately the `interface SqlInterruptHandle` in UDL is in reference to the rust side. So I believe the three options we have are:
1. Rename SqlInterrupt handle in the support component (used quite a bit though)
2. Typealias in our kotlin/swift wrappers to keep the name (though still leaks a tiny bit)
3. Keep same

I have no strong preference either way but would probably lean to leaving as-is if only to keep things simple for now until the interrupt rework takes place.